src/libs/PolicyUtils.ts

```diff
@@ -110,7 +110,7 @@ function isExpensifyGuideTeam(email: string): boolean {
 /**
  * Checks if the current user is an admin of the policy.
  */
-const isPolicyAdmin = (policy: OnyxEntry<Policy>): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
+const isPolicyAdmin = (policy?: OnyxEntry<Policy>): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
```

Comment:
```suggestion
const isPolicyAdmin = (policy: OnyxEntry<Policy> | undefined): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
```

This way we are required to pass something, even if it's undefined

Reply:
Hey @shahinyan11 how about this?

Reply:
Done